dftimewolf/cli/recipes/grr_fetch_files.py

```diff
@@ -0,0 +1,42 @@
+"""Fetches specific files from a GRR host."""
+from __future__ import unicode_literals
+
+_short_description = 'Fetches specific files from a GRR host.'
```

Comment:
This works for fetching files from multiple hosts too, please fix

Reply:
Done.